__tests__/integration.test.js

```diff
@@ -29,11 +29,32 @@ describe("/api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then(({ body }) => {
-        expect(body.length).toBe(3);
         body.forEach((topic) => {
           expect(typeof topic.description).toBe("string");
           expect(typeof topic.slug).toBe("string");
         });
       });
   });
 });
+
+describe("/api", () => {
+  test("GET 200 - Responds with all api endpoints and information about them.", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Object.keys(body)).toMatchObject([
```

Comment:
We want this test to be dynamic - we dont want to have to keep going back to this test and adding in new endpoints every time we update our endpoints.json file and this test will break if we don't. instead we can directly require in our endpoints.json file here and test the output against that - the endpoints.json file will always be up to date.